dataclients/kubernetes/kube.go

```diff
@@ -1193,6 +1192,7 @@ func computeBackendWeights(backendWeights map[string]float64, rule *rule) {
 			if weight > 0 {
 				sc.lastActive = path.Backend
 			}
+			sc.weightsCount++
```

Comment:
Probably better to only add this if `weight > 0` because we only need to create `noop` predicates for those cases.

Reply:
fixed